### PR TITLE
map tools: data verification tool, commit build files and proto files

### PR DESCRIPTION
**Pull request by ooooooops**
map tools: data verification tool, commit build files and proto files

### CODE REVIEW

File: modules/map/tools/map_datachecker/client/BUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+cc_binary(
+    name = "map_datachecker_client",
+    deps = [
+        ":map_datachecker_client_lib",
+    ],
+)
+
+cc_library(
+    name = "map_datachecker_client_lib",
+    srcs = glob(["*.cc"]),
```
Comment (jinghaomiao) [inline modules/map/tools/map_datachecker/client/BUILD]: 每一个.cc需要是单独的target

File: modules/map/tools/map_datachecker/server/BUILD [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+cc_binary(
+    name = "map_datachecker_server",
+    deps = [
+        ":map_datachecker_server_lib",
+    ],
+)
+
+cc_library(
+    name = "map_datachecker_server_lib",
+    srcs = glob(["*.cc"]),
```
Comment (jinghaomiao) [inline modules/map/tools/map_datachecker/server/BUILD]: 同理

File: modules/map/tools/map_datachecker/client/BUILD (hunk 1 above)
Comment (ooooooops) [reply modules/map/tools/map_datachecker/client/BUILD]: thanks, done.

File: modules/map/tools/map_datachecker/server/BUILD (hunk 2 above)
Comment (ooooooops) [reply modules/map/tools/map_datachecker/server/BUILD]: thanks, done.

**Pull request merged**